tests/std/include/range_algorithm_support.hpp

```diff
@@ -510,17 +510,8 @@ namespace test {
         range() = default;
         constexpr explicit range(span<Element> elements) noexcept : elements_{elements} {}
 
-        range(const range&) requires derived_from<Category, fwd> = default;
-        range& operator=(const range&) requires derived_from<Category, fwd> = default;
-
-        constexpr range(range&& that) noexcept
-            : elements_{exchange(that.elements_, {})}, begin_called_{that.begin_called_} {}
-
-        constexpr range& operator=(range&& that) noexcept {
-            elements_     = exchange(that.elements_, {});
-            begin_called_ = that.begin_called_;
-            return *this;
-        }
+        range(const range&) = delete;
+        range& operator=(const range&) = delete;

```

Comment:
This appears to be one of your east-`const` files.

Reply:
Yes - I've been writing `const`-west in more recent tests since we had that discussion (in an issue I don't recall at the moment) about improving overall consistency in the test codebase now that our developer base is growing. I'll switch this file if I can ever get it to settle down to the point where I don't have modifications in multiple active simultaneous PRs.